storage/ResidentDatabaseSchema.ts

```diff
@@ -57,7 +57,7 @@ type ResidentDatabaseSchema = NamedSchema<
         hasCarer: string;
         type: string;
         isLiveIn: string;
-        liveInStartDate: { month?: number; year?: number };
+        liveInStartDate: string;
```

Comment:
This change should really have a data migration added for it, or you'll have issues with older processes.

Reply:
I'm not sure how to do this.

Reply:
I'm not sure how to do this.

Reply:
Bump the database schema version, and add a migration to `storage/migrations/data` (and don't forget to set it up in the index file). Look at previous migrations for hints.